gremlin-go/driver/transporterFactory.go

```diff
@@ -0,0 +1,37 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package gremlingo
+
+// TransporterType is an alias for valid transport protocols.
+type TransporterType int
+
+const (
+	// Gorilla transport layer: github.com/gorilla/websocket
+	Gorilla TransporterType = iota
```

Comment:
as a general style guide, let us start enums from 1 because variables have a 0 default value and hence starting an enum from 1 will help distinguish between uninitialized enum vs. intentional value of an enum

Reply:
Given the way we consume configuration, it wouldn't be an issue in this case.

Regardless, this is a good suggestion and has been changed for MS2!